### PR TITLE
Implemented Social Media Buttons

### PR DESCRIPTION
**Pull request by stevenpi**
Implemented the social media buttons using `Shariff` since it was part of the issue

Screenshot of implementation in the UI:  
socialmediabuttons_feature

### CODE REVIEW

File: Blog/urls.py [hunk 1]
```diff
@@ -12,4 +12,3 @@
     # url for inserting into the db (lookup on views.py)
     # path('insert/', views.insert_to_db),
 ]
```
Comment (stevenpi) [inline Blog/urls.py]: I have no clue how this happened, the line is still there locally.

Comment (mbaechtold) [reply Blog/urls.py]: You could remove this unused code in a commit of its own.

Comment (stevenpi) [reply Blog/urls.py]: Sorry there is no need to remove it -> this code is part of the script which adds many entities to the database. I should consider to read my own comments.

**Comment by stevenpi** [comment]
Added the screenshot as requested

**Pull request merged**